include/avb_verify.h

```diff
@@ -98,4 +99,17 @@ static inline int get_boot_device(AvbOps *ops)
 	return -1;
 }
 
+static inline struct blk_desc *get_blk(AvbOps *ops)
+{
+	struct AvbOpsData *data;
+
+	if (ops) {
```

Comment:
you can simplify this to `if (ops && ops->user_data) return ops->userdata->blk`.
It is easier to read. Or can leave it as is